docs/packaging.md

```diff
@@ -0,0 +1,53 @@
+# Application Lifecycle
```

Comment:
I don't see that much duplicate, though. I suggest linking to the given doc for the duplicate part and keep the packaging  and EC2 deployment here.